rpc.go

```diff
@@ -1,7 +1,28 @@
 package rpc
 
-func Ping(params interface{}, id ...interface{}) string {
+import (
+	"encoding/json"
+)
+
+// RPC struct
+type RPC struct {
+	base
+}
+
+// NewRPC is a constructor of RPC
+func NewRPC(node string) RPC {
+	return RPC{base: newBase(node, nil)}
+}
+
+// Ping sends request to node
+func (rpc RPC) Ping(params interface{}, id interface{}) {
 	const method = "ping"
-	return call(callInterface{node: "https://corgi-rpc.codechain.io/", method: method, id: id})
-	// TODO: Use node uri from environment
+	rpc.call(callInterface{method: method, id: id})
+}
+
+// Version checking
+func (rpc RPC) Version(params interface{}, id interface{}) string {
+	const method = "version"
+	res, _ := json.Marshal(rpc.call(callInterface{method: method, id: id}))
```

Comment:
What is the role of json.Marshal here?

Reply:
I designed `rpc.call` to return `Result` type. In order to convert `Result` to `string`, `json.Marshal` is necessary. 

Another option is to make `rpc.call` to return `string`
The type of result of rpc flows `[]byte` => `Result` => `string` in current design.
If `json.Marshal` looks redundant, I can do this with `[]byte` => `string`
What do you suggest?

Reply:
The returned string is `{"jsonrpc":"2.0","result":"1.4.0-alpha.3"}`.
However, it should be "1.4.0-alpha.3". Could you fix it just return the version string?

Reply:
https://github.com/CodeChain-io/codechain-rpc-go/pull/4/commits/a0aa9203e181a6529130987945c9db67bb696f6d